openlibrary/core/bookshelves_events.py

```diff
@@ -17,6 +19,20 @@ class BookshelvesEvents(db.CommonExtras):
     TABLENAME = 'bookshelves_events'
     NULL_EDITION_ID = -1
 
+    @classmethod
+    def summary(cls):
+        return {
+            'total_yearly_reading_goals': {
+                'total': BookshelvesEvents.total_yearly_reading_goals(),
+                'month': BookshelvesEvents.total_yearly_reading_goals(
+                    since=DATE_ONE_MONTH_AGO
+                ),
+                'week': BookshelvesEvents.total_yearly_reading_goals(
+                    since=DATE_ONE_WEEK_AGO
+                ),
+            },
+        }
```

Comment:
Could you move this and the `total_yearly_reading_goals` function to https://github.com/internetarchive/openlibrary/blob/master/openlibrary/core/yearly_reading_goals.py?